cmd/transform/transform.go

```diff
@@ -87,6 +90,14 @@ func (o *Options) run() error {
 		return err
 	}
 
+	var ignoredPatchesDir string
+	if len(o.IgnoredPatchesDir) > 0 {
```

Comment:
minor minor nit: I usually use would use `o.IgnoredPatchesDir != ""`. Don't know if it is that big of a deal just pointing it out.

Reply:
I'm happy to change. I've used both in the past (golang's lack of a nil value for strings is annoying), and I think I actually use `!=0` more often. I can change this instance.

Reply:
changed it.